### ISSUE TITLE
LR / BSZ grid for MoE reference model

### ISSUE THREAD

**Issue by aaronkl** [OpenEuroLLM/Taskboard#162]
Find a good configuration for learning rate and global batch size for the Qwen3 0.6BA100M model.

**Comment by aaronkl** [comment]
Validation loss of a HP grid for the Qwen3 0.6BA100M model trained for 30B tokens. Training curves look smooth, no spikes or instabilities.

<img width="1200" height="800" alt="Image" src="https://github.com/user-attachments/assets/f5730df0-7971-49d0-936f-bfa82108bea3" />


Next step: I will re-run the grid (or a subset) with a MoE config that is closer aligned to our current dense reference configs
